zerver/tests/test_outgoing_webhook_system.py

```diff
@@ -29,43 +31,54 @@ def timeout_error(http_method, final_url, data, **request_kwargs):
     # type: (Any, Any, Any, Any) -> Any
     raise requests.exceptions.Timeout
 
+class MockServiceHandler(object):
```

Comment:
Yeah `MockServiceHandler(OutgoingWebhookServiceInterface)` would be better. `process_command` method wasn't being used here, so I had to remove it else I was getting backend test coverage errors.